sys-process/audit/audit-3.0.6.ebuild

```diff
@@ -182,11 +178,12 @@ pkg_postinst() {
 lockdown_perms() {
 	# Upstream wants these to have restrictive perms.
 	# Should not || die as not all paths may exist.
-	# Flatcar: No lockdown of permissions - it's probably only
-	# related to auditd.
-	# local basedir="${1}"
+	# Flatcar: We don't include ausearch and aureport
+	# so they're removed from the hardening list
+	local basedir="${1}"
 	# chmod 0750 "${basedir}"/sbin/au{ditctl,ditd,report,search,trace} 2>/dev/null
-	# chmod 0750 "${basedir}"/var/log/audit 2>/dev/null
-	# chmod 0640 "${basedir}"/etc/audit/{auditd.conf,audit*.rules*} 2>/dev/null
+	chmod 0750 "${basedir}"/sbin/au{ditctl,ditd,trace} 2>/dev/null
+	chmod 0750 "${basedir}"/var/log/audit 2>/dev/null
+	chmod 0640 "${basedir}"/etc/audit/{auditd.conf,audit*.rules*} 2>/dev/null
```

Comment:
Thanks for the changes. I think the location of the config and the default permissions will be a bit tricky. Further above the location used is ` /usr/share/audit/rules.d` which is due to the auto update only covering the `/usr` partition and not `/etc` or `/var`. If we need files under `/etc` or `/var` we normally set them up through systemd-tmpfile directives to ensure that they exist on updated nodes, too. One possible solution for the config file would be to place the config file `/etc/audit/auditd.conf` under `/usr` and  set up symlinks from `/etc/` to it which is currently done for the rules under `/etc/audit/rules.d/` already (see `files/audit-rules.tmpfiles`).

Reply:
(The user would still be able to customize the config by placing a regular file there and systemd-tmpfiles would not touch it to create a symlink)

Reply:
E.g., for the log directory it would be an entry like `d   /var/log/audit        0750    -   -   -   -`

Reply:
If the `auditd.conf` file actually just holds the default settings we can even try to delete it

Reply:
I tried to use it without the file and it somehow worked, just an error message in the log but it seemed to do what it should, no idea.
Anyway, I've now started a test build of the current state which I think is very good. When that's done we can add a changelog entry and that's it.